presto-main/src/test/java/com/facebook/presto/security/TestFileBasedSystemAccessControl.java

```diff
@@ -533,7 +537,32 @@ private AccessControlManager newAccessControlManager(TransactionManager transact
 
     private String getResourcePath(String resourceName)
```

Comment:
All these are used in copy(source, destination). I believe the code can be simplified if we have a method for copy and if there is an exception, fall into this.

Reply:
@arunthirupathi Refactor the code a bit for a mode concise implementation of getResourceFile() and get rid of getResourcePath(). Please see if you have further comments